homeassistant/components/lawn_mower/__init__.py

```diff
@@ -84,11 +84,9 @@ class LawnMowerEntity(Entity, cached_properties=CACHED_PROPERTIES_WITH_ATTR_):
 
     @final
     @property
-    def state(self) -> str | None:
+    def state(self) -> LawnMowerActivity | None:
```

Comment:
In valve we still annotate this as `str` and not an enum.

Reply:
True, I'll change that.